go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resourcehealth/armresourcehealth v1.3.0
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v1.2.0
 	github.com/Azure/azure-sdk-for-go/sdk/tracing/azotel v0.4.0
-	github.com/Azure/azure-service-operator/v2 v2.11.0
+	github.com/Azure/azure-service-operator/v2 v2.12.1-0.20250331045328-e711390fd009
```

Comment:
Sorry I dont fully understand this. 
Is this commit `e711390fd009` referring to the v2/go.mod in ASO with `sigs.k8s.io/controller-runtime` updated to `v0.20.4` ? 
If so, could you please mention how we got this commit tag ?

Reply:
I think this is referring to a commit after v2.12. It was pulled in by `go get -u` and things compiled, so I've left it intact, but I assume we want to replace it with v2.13 when that is released.